index.html

```diff
@@ -7,93 +7,211 @@
+                                <li>Codepen</li>
+                                <li>Terminal</li>
+                            </ul>
+                        </div>
                     </div>
-                </div>
 
-                <hr class="divider">
+                    <hr class="divider">
 
-                <div class="spprtItems flex column">
-                    <div class="socialMedia flex">
-                        <a class="github flex" role="button" href=""><img src="icons&imgs/Icon-GitHub.png"
-                                alt="github"></a>
+                    <div class="spprtItems flex column">
+                        <div class="socialMedia flex">
+                            <a class="github flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Icon-GitHub.png" alt="github"></a>
 
-                        <a class="linkedin flex" role="button" href=""><img src="icons&imgs/Icon-Linkedin.png"
-                                alt="linkedin"></a>
+                            <a class="linkedin flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Icon-Linkedin.png" alt="linkedin"></a>
 
-                        <a class="angelist flex" role="button" href=""><img src="icons&imgs/angelist.png"
-                                alt="angelist"></a>
+                            <a class="angelist flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/angelist.png" alt="angelist"></a>
 
-                        <a class="twitter flex" role="button" href=""><img src="icons&imgs/twitter.png"
-                                alt="twitter"></a>
+                            <a class="twitter flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/twitter.png" alt="twitter"></a>
 
-                        <a class="medium flex" role="button" href=""><img src="icons&imgs/Vector.png" alt="medium"></a>
+                            <a class="medium flex" role="button" href=""><img class="comunic"
+                                    src="icons&imgs/Vector.png" alt="medium"></a>
+                        </div>
+                        <div class="elementRight flex">
+                            <img src="icons&imgs/Group55.png" alt="arrowgroup55">
+                            <p class="work">Work</p>
+                        </div>
                     </div>
-                    <div class="elementRight flex">
-                        <img src="icons&imgs/Group55.png" alt="arrowgroup55">
-                        <p class="work">Work</p>
+                </div>
+            </section>
+
+        </div>
+        <div class="sub-container-2 flex column">
+            <section class="recentwork flex column">
+                <div class="h flex column">
+                    <h1 class="mainTxt-2">MY RECENT WORK</h1>
```

Comment:
- [ ] Please know that you should only use one `h1` tag in your webpage, and in your case you already did that with the main title (which is good), kindly change it to another `hx` tag. 😁